src/screens/FeedScreen/Header.js

```diff
@@ -252,7 +252,7 @@ const _renderProfileNormal = ({
         refSheet.current.close();
         onDeletePost();
       },
-      style: {color: colors.red}
+      style: {color: COLORS.red}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been updated from `colors.red` to `COLORS.red`. This change is consistent with the update in the import statement.
```diff
- style: {color: colors.red}
+ style: {color: COLORS.red}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->